app/serializers/review_serializer.rb

```diff
@@ -0,0 +1,5 @@
+class ReviewSerializer < ActiveModel::Serializer
+  attributes :id, :title, :body
+
+  belongs_to :artist
```

Comment:
careful, you actually probably dont want line 4, because then we end up having every review mention its artist, and it becomes a bad loop